addons/web/static/src/core/model_field_selector/model_field_selector_popover.js

```diff
@@ -220,7 +220,7 @@ export class ModelFieldSelectorPopover extends Component {
         }
         this.keepLast.add(Promise.resolve());
         this.state.page.selectedName = field.name;
-        this.props.update(this.state.page.path);
+        this.props.update(this.state.page.path, field, this.state.page.fieldDefs);
```

Comment:
The parent should be able to get all the infos via the path and the hook "useLoadPathDescription"

Reply:
Nice, I will change this